Branch-SDK/Branch-SDK/BNCDeviceInfo.m

```diff
@@ -169,9 +170,9 @@ + (NSString*) userAgentString {
             DISPATCH_BLOCK_DETACHED | DISPATCH_BLOCK_ENFORCE_QOS_CLASS,
             QOS_CLASS_USER_INTERACTIVE,
             0,  ^ {
-                //NSLog(@"Will userAgent.");
+                BNCLogDebug(@"Will userAgent.");
                 setBrowserUserAgent();
-                //NSLog(@"Did  userAgent.");
+                BNCLogDebug(@"Did  userAgent.");
```

Comment:
Also, if we're logging something for other people, we should definitely be way more descriptive in the message.